I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/scripts/overview.rpy", line 639, in script call
    call time_event_check from new_day_2
  File "game/scripts/daily_check.rpy", line 111, in script call
    call call_available_event(time_check_events, 0, True) from time_event_check_2
  File "game/scripts/event.rpy", line 1306, in script
    $ events_list = event_storage.get_available_events(priority, **kwargs)
  File "game/scripts/event.rpy", line 1306, in <module>
    $ events_list = event_storage.get_available_events(priority, **kwargs)
  File "game/scripts/event.rpy", line 493, in get_available_events
    if event.is_available(**kwargs):
  File "game/scripts/event.rpy", line 1165, in is_available
    if not condition.is_fulfilled(**kwargs):
  File "game/scripts/conditions.rpy", line 2731, in is_fulfilled
    if time.compare_today(10, 1, 2023) == -1 and is_intro:
NameError: name 'is_intro' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/scripts/overview.rpy", line 639, in script call
    call time_event_check from new_day_2
  File "game/scripts/daily_check.rpy", line 111, in script call
    call call_available_event(time_check_events, 0, True) from time_event_check_2
  File "game/scripts/event.rpy", line 1306, in script
    $ events_list = event_storage.get_available_events(priority, **kwargs)
  File "D:\RenPy\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\RenPy\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/scripts/event.rpy", line 1306, in <module>
    $ events_list = event_storage.get_available_events(priority, **kwargs)
  File "game/scripts/event.rpy", line 493, in get_available_events
    if event.is_available(**kwargs):
  File "game/scripts/event.rpy", line 1165, in is_available
    if not condition.is_fulfilled(**kwargs):
  File "game/scripts/conditions.rpy", line 2731, in is_fulfilled
    if time.compare_today(10, 1, 2023) == -1 and is_intro:
NameError: name 'is_intro' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.1.3.23091805
Mind the School 0.1.3
Wed Apr  3 18:22:58 2024
